ui/src/components/SUGGESTION_BOX.tsx

```diff
@@ -1,16 +1,17 @@
+import React from "react";
 import { Typography, Input } from "antd";
 import "./SUGGESTION_BOX.css";
 
 const { Title } = Typography;
 const { TextArea } = Input;
 
-const handleKeyPress = (ev: React.KeyboardEvent<HTMLTextAreaElement>) => {
-  console.log("handleKeyPress", ev);
-};
-
 function SUGGESTION_BOX() {
+  const handleKeyPress = (ev: React.KeyboardEvent<HTMLTextAreaElement>) => {
+    console.log("handleKeyPress", ev);
+  };
+
```

Comment:
actually we don't need this, it does nothing

Reply:
The suggestion box does not send yet anything anywhere, but we need it later. We discuss this with Maria.